tests/helpers/test_reports_helpers.py

```diff
@@ -74,35 +83,42 @@ def test_delete_reports(app, freezer):
         assert os.path.isfile(f"{app.config['REPORTS_DIR']}/{filename}") is False
 
 
-def test_get_cherrypicked_samples(app, freezer):
+# ----- get_cherrypicked_samples tests -----
 
-    expected = pd.DataFrame(
-        ["MCM001", "MCM003", "MCM005"], columns=[FIELD_ROOT_SAMPLE_ID], index=[0, 1, 2]
-    )
+
+def test_get_cherrypicked_samples_no_beckman(app, freezer):
+    expected = [
+        pd.DataFrame(
+            ["MCM001", "MCM003", "MCM005"], columns=[FIELD_ROOT_SAMPLE_ID], index=[0, 1, 2]
+        ),  # Sentinel query response
+        pd.DataFrame([]),  # Beckman query response
+    ]
     samples = ["MCM001", "MCM002", "MCM003", "MCM004", "MCM005"]
     plate_barcodes = ["123", "456"]
 
     with app.app_context():
         with patch("sqlalchemy.create_engine", return_value=Mock()):
             with patch(
                 "pandas.read_sql",
-                return_value=expected,
+                side_effect=expected,
             ):
                 returned_samples = get_cherrypicked_samples(samples, plate_barcodes)
                 assert returned_samples.at[0, FIELD_ROOT_SAMPLE_ID] == "MCM001"
                 assert returned_samples.at[1, FIELD_ROOT_SAMPLE_ID] == "MCM003"
                 assert returned_samples.at[2, FIELD_ROOT_SAMPLE_ID] == "MCM005"
 
 
-def test_get_cherrypicked_samples_chunking(app, freezer):
-
-    # Note: This represents the results of three different database queries
-    # each of which gets indexed from 0. Do not changes the indicies here unless
-    # you have modified the behaviour of the query.
+def test_get_cherrypicked_samples_chunking_no_beckman(app, freezer):
+    # Note: This represents the results of three different (Sentinel, Beckman) sets of
+    # database queries, each Sentinel query getting indexed from 0. Do not changes the
+    # indicies here unless you have modified the behaviour of the query.
     query_results = [
         pd.DataFrame(["MCM001"], columns=[FIELD_ROOT_SAMPLE_ID], index=[0]),
+        pd.DataFrame([]),
```

Comment:
maybe add the beckman comment for these empty dataframes so we know why they are here like above